Adaptors/QueueCommon/src/Amqp.cs

```diff
@@ -36,4 +36,5 @@ public class Amqp
   public bool   AllowHostMismatch { get; set; }
   public int    MaxRetries        { get; set; }
   public int    LinkCredit        { get; set; }
+  public int    ParallelismLimit  { get; set; } = 200;
```

Comment:
That is very high. I don't think it is a good idea. You can put 0 to use the number of threads by default.